### PR TITLE
Branding update changes

### LINKED ISSUE #69: Update to comply with the New NSF Branding Theme
Update this site to reflect the NSF branding policy changes (e.g. NSF NCAR) wherever applicable, e.g. logos, text, etc.

### PR DESCRIPTION
**Pull request by erogluorhan**
Closes #69. 

- This PR did not touch the older-than-branding-update assets such as the blog posts. 
- Also, the ones in the site header ill need to be fixed by the [ncar-koru-jekyll theme](https://github.com/NCAR/koru-jekyll)

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ youtubeID: 34zFGkDwJPc
 # Welcome
 
 **Geo**science **C**ommunity **A**nalysis **T**oolkit (**GeoCAT**), established in 2019, leads the software
-engineering efforts of the National Center for Atmospheric Research (NCAR) Computational and Information
+engineering efforts of the NSF National Center for Atmospheric Research (NSF NCAR) Computational and Information
 Systems Lab (CISL)’s “Pivot to Python” initiative. Support for GeoCAT is provided by the U.S. **National
```
Comment (cyschneck) [inline index.md]: The U.S is part of the full name `**U.S. National Science Foundation**`

**Review by cyschneck — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (erogluorhan) [reply index.md]: Good catch, done!

**Review by cyschneck — APPROVED** [review]

**Review by kafitzgerald — APPROVED** [review]
Looks good!

Do we need to log an issue on the template repo or is that already communicated?

**Comment by erogluorhan** [comment]
> Looks good!
> 
> Do we need to log an issue on the template repo or is that already communicated?

Yeah, I've created [that](https://github.com/NCAR/koru-jekyll/issues/16)

**Pull request merged**